packages/@react-spectrum/tooltip/src/TooltipTrigger.tsx

```diff
@@ -30,7 +31,22 @@ function TooltipTrigger(props: SpectrumTooltipTriggerProps) {
     offset = DEFAULT_OFFSET
   } = props;
 
-  let [trigger, tooltip] = React.Children.toArray(children);
+  let tooltipRef = useRef();
+
+  let [trigger, tooltip] = React.Children.toArray(children) as [ReactElement, ReactElement];
+
+  // this does mean that we can't have some element wrapping the Tooltip
+  // we'd need to do it through context if we wanted to allow for that
+  let tooltipWithRef = React.cloneElement(tooltip, {
```

Comment:
Yeah we should avoid `cloneElement`. Someone might do something like `<MyFancyTooltip>` that wraps `<Tooltip>` and not pass through the ref...